lib/compilers/glimmer/index.js

```diff
@@ -42,3 +57,26 @@ export function getTemplateSpecifier(rootName, relativePath) {
 
   return `template:/${rootName}/${pathParts.join('/')}`;
 }
+
+function getManagerIdFromPath(path) {
+  let parts = path.split('.');
+  let managerId = null;
+
+  if (parts.length > 2) {
+    let lastIndex = parts.length - 1;
+    managerId = parts.splice(lastIndex - 1, 1)[0];
+  }
+
+  return managerId;
+}
+
+function removeManagerIdFromPath(path) {
```

Comment:
I don't generally think we need this. I'd much prefer to leave the modules on disk where they were to "shifting" their positions...